content/tutorials/terrain_and_DEMs/GRASS_terrain.qmd

```diff
@@ -0,0 +1,1085 @@
+:::
+
+::: g-col-6
+![flow accumulation map for Flagstaff region](img_terrain/watershed_accumulation.webp)  
+
+![schematic diagram of multiple flow direction flow accumulation model on a DEM](img_terrain/watershed.webp)
+:::
+:::::
+
+As with slope, a legend can help convey the information of a flow accumulation map more clearly. The distribution of accumulation values is non-linear, however, with many cells on hillslopes having minimal accumulation and a few cells in the bottoms of major streams having much higher values. For this reason, a logarithmic scale better communicates the relationships between colors and values. Create a legend as described previously but with some additional options.
+
+::::::::::::::: {.panel-tabset group="language"}
+
+#### GUI
+
+-  Name of raster map: watershed_accumulation10000@terrain (Input tab).  
+
+-  Legend title: "Flow Accumulation" (Title tab).  
+
+-  Title font size: 14 (Title tab).  
+
+-  Use logarithmic scale: check the box (Advanced tab).  
+
+-  Specific values to draw ticks: 1,10,100,1000,10000,100000,1000000 (Gradient tab).  
+
+-  Font size: 12 (Font settings tab).  
+
+#### Command line
+
+```{python}
+d.legend -l raster=watershed_accumulation100000 title="Flow Accumulation" title_fontsize=14 label_values=1,10,100,1000,10000,100000,1000000 fontsize=12
+```
+
+#### Python
+
+```{python}
+gs.run_command("d.legend", 
+                raster="watershed_accumulation10000", 
+                title="Flow Accumulation", 
+                title_fontsize=14, 
+                label_values="1,10,100,1000,10000,100000,1000000", 
+                fontsize=12, 
+                flags="l")
+```
+
+:::::::::::::::
+
+Here is a flow accumulation map with a legend using a logarithmic scale. Note higher accumulation on the slopes of the San Francisco peaks, in major stream courses, and in small, shallow lake depressions.  
+
+![flow accumulation map coloring a relief map with logarithmic scale legend](img_terrain/watershed_accumulation_legend.webp)
+
+## Stream networks
+
+*r.watershed* also produced a maps of streams, representing the primary drainages of each 10,000-cell watershed basin.  
+
+::::: grid
+::: g-col-6
+
+-  It is a raster map of streams with each stream cell coded and colored to match the basin that it drains, seen in the map of the Flagstaff region to the right.  
+
+-  The background is set to black in the display properties because it is otherwise very difficult to distinguish the raster stream cells.  
+:::
+
+::: g-col-6
+![raster stream segments map for each watershed basin](img_terrain/watershed_streams.webp)  
+:::
+:::::
+
+### Vectorizing stream network with *r.to.vect*  
+
+We can make the streams more easily visible if we convert the raster stream map to a vector line map. This can be done easily with the [r.to.vect](https://grass.osgeo.org/grass84/manuals/r.to.vect.html) tool that converts rasters maps into vectors.  
```

Comment:
```suggestion
We can make the streams more easily visible if we convert the raster stream map to a vector line map. This can be done easily with the [r.to.vect](https://grass.osgeo.org/grass-stable/manuals/r.to.vect.html) tool that converts rasters maps into vectors.  
```